domains/sprit.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Codedspeit's personal note site",
+  "repo": "https://github.com/codedsprit/spritnote",
+  "owner": {
+    "username": "codedsprit",
+    "email": "roshan0x01@gmail.com"
+  },
+  "record": {
+    "CNAME": "codedsprit.github.io/spritnote"
```

Comment:
CNAME can not contain `/`

Reply:
It's not changed to netlify, which doesn't contain ``/``, ``//``.